glommio/src/channels/mod.rs

```diff
@@ -146,6 +146,44 @@ pub mod local_channel;
 /// [`StreamExt`]: https://docs.rs/futures-lite/1.11.1/futures_lite/stream/trait.StreamExt.html
 pub mod shared_channel;
 
+/// A channel mesh consists of a group of participating executors (peers) and the channels between
+/// each pair of them.
+///
+/// Within a mesh, peers are identified by unique peer ids, which are used to specify the
+/// source/destination which messages are sent to/received from. Peer ids are assigned when the last
+/// peer joins by the indexes to the list of all peers, which is sorted by their executor ids. So
+/// multiple meshes can be built over the same set of peers, and the peer ids are guaranteed to be
```

Comment:
but does that mean that all meshes have to be built at once too ?

Reply:
Meshes could be built at any time. Any peer will have the same peer id across all meshes, as long as they have the same set of peers, because peer id is determined by the sorted order of its executor id.